resources/qml/PrinterSelector/MachineSelectorList.qml

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.7
+import QtQuick.Controls 2.3
+
+import UM 1.2 as UM
+import Cura 1.0 as Cura
+
+Column
+{
+    id: machineSelectorList
+
+    Label
+    {
+        text: catalog.i18nc("@label", "Network connected printers")
+        visible: networkedPrintersModel.items.length > 0
+        leftPadding: UM.Theme.getSize("default_margin").width
+        height: visible ? contentHeight + 2 * UM.Theme.getSize("default_margin").height : 0
+        renderType: Text.NativeRendering
+        font: UM.Theme.getFont("medium")
+        color: UM.Theme.getColor("text_medium")
+        verticalAlignment: Text.AlignVCenter
+    }
+
+    Repeater
+    {
+        id: networkedPrinters
+
+        model: UM.ContainerStacksModel
+        {
+            id: networkedPrintersModel
+            filter: {"type": "machine", "um_network_key": "*", "hidden": "False"}
```

Comment:
{ needs to be on newline.

Reply:
Done in 21bfa6e4c20819e09b8cb2f27f62427dc71c8476